lib/matplotlib/projections/polar.py

```diff
@@ -1171,24 +1169,73 @@ def get_theta_direction(self):
         return self._direction.get_matrix()[0, 0]
 
     def set_rmax(self, rmax):
+        """Update the maximum radius
+
+        Parameters
+        ----------
+        rmax : number
+            Set maximum radius to value rmax.
+        """
         self.viewLim.y1 = rmax
 
     def get_rmax(self):
+        """
+        Returns
+        -------
+        float
+            The maximum radius of chart.
+        """
         return self.viewLim.ymax
 
     def set_rmin(self, rmin):
+        """Updates the minimum radius
+        Parameters
+        ----------
+        rmin : number
+            Set minimumradius to value rmin.
+        """
         self.viewLim.y0 = rmin
 
     def get_rmin(self):
+        """
+        Returns
+        -------
+        float
+            The minimum radius of chart.
```

Comment:
I don't think we've ever used the word 'chart' to describe our plots before (except when referring to a _specific_ type of plot, like a pie chart.) This could be something like "the minimum data radius" or something that matches the Cartesian form, but in any case, I don't think we should use 'chart' for these.

Reply:
I think we use “polar plot” and “radar chart”.